src/Microsoft.Azure.WebJobs.Extensions.Kafka/Trigger/KafkaTriggerAttributeBindingProvider.cs

```diff
@@ -100,10 +100,28 @@ private KafkaListenerConfiguration CreateConsumerConfiguration(KafkaTriggerAttri
             {
                 consumerConfig.SaslPassword = this.config.ResolveSecureSetting(nameResolver, attribute.Password);
                 consumerConfig.SaslUsername = this.config.ResolveSecureSetting(nameResolver, attribute.Username);
-                consumerConfig.SslKeyLocation = this.config.ResolveSecureSetting(nameResolver, attribute.SslKeyLocation);
                 consumerConfig.SslKeyPassword = this.config.ResolveSecureSetting(nameResolver, attribute.SslKeyPassword);
-                consumerConfig.SslCertificateLocation = this.config.ResolveSecureSetting(nameResolver, attribute.SslCertificateLocation);
-                consumerConfig.SslCaLocation = this.config.ResolveSecureSetting(nameResolver, attribute.SslCaLocation);
+
+                var sslKeyLocation = this.config.ResolveSecureSetting(nameResolver, attribute.SslKeyLocation);
+                if (!AzureFunctionsFileHelper.TryGetValidFilePath(sslKeyLocation, out var resolvedSslKeyLocation))
+                {
+                    resolvedSslKeyLocation = sslKeyLocation;
+                }
+
+                var sslCertificateLocation = this.config.ResolveSecureSetting(nameResolver, attribute.SslCertificateLocation);
+                if (!AzureFunctionsFileHelper.TryGetValidFilePath(sslCertificateLocation, out var resolvedSslCertificationLocation))
+                {
+                    resolvedSslCertificationLocation = sslCertificateLocation;
+                }
+
+                var sslCaLocation = this.config.ResolveSecureSetting(nameResolver, attribute.SslCaLocation);
+                if (!AzureFunctionsFileHelper.TryGetValidFilePath(sslCaLocation, out var resolvedSslCaLocation))
```

Comment:
I see that the TryGetValidFilePath adds the base function folder to the relative path. This would affect the path in both Public Azure and in case of containers. Please verify once for the container scenario as well.

Reply:
Done. Verified for both cases.